ROADMAP.md

```diff
@@ -1,39 +1,42 @@
 # Roadmap
-This document represents the current working roadmap for Pantheon.  It is a living document, which will evolve and change over time.  In particular the features in later versions are likely to be refined and change.
+This document represents the current working roadmap for Pantheon.  It is a living document, which will 
+evolve and change over time.  In particular the features in later versions are likely to be refined and change.
 
-We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  Our now scale is about 3 months, next about 6 months, and then later is beyond.
+We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  
+Our now scale is about 3 months, next about 6 months, and then later is beyond.
 
-## Now
-Our key three areas for now are:
-* Permissioning
+## Now/v1.1
+Our key areas for now are:
+* Privacy
+* Smart Contract Based Permissioning
 * First Class Client
-* iBFT 2.0
 
-### Permissioning
-We are implementing the key elements of an Enterprise Ethereum Permissioned network.  The initial version of this will be based around a JSON RPC API to manage the network. This will form the foundation for a smart contract based solution which will be developed in the `next` release (1.1)
+### Privacy
+The Enterprise Ethereum `restricted` privacy will be implemented.
+
+### Smart Contract Based Permissioning
+Building on the Permissioning system  implemented in version 1.0 of Pantheon, we will use a smart contract to share this information across the network, giving a consistent set of permissions, and ensuring that all nodes in the network work consistently.
```

Comment:
How about:
"Building on the Permissioning system implemented in version 1.0 of Pantheon, we will use a smart contract to share this information across the network, creating a single, decentralized source of truth for node and account level permission data."

Reply:
Updated